src/Core/Framework/Adapter/Command/MediaPublishCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Adapter\Command;
+
+use League\Flysystem\FilesystemInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Finder\Finder;
+
+class MediaPublishCommand extends Command
+{
+    protected static $defaultName = 'media:publish';
```

Comment:
When we make such a new command. We should make it aware of multiple storage. Like pushing from sftp to s3.

Reply:
That's a very advanced use case. This command is only intended for people migrating from local to a remote filesystem. Migrating from one remote filesystem to another is highly unlikely and not that easy as scanning for files 😅

Reply:
Thats the reason behind why we don't wanted such command. Normally the usage of an cli tool for the specific provider is always much better .)

Reply:
Whats your opionon @janbuecker ?

Reply:
We have (or had) such command in Shopware 5 and I would never do I again. CLI tools from the provider itself are way better and perform better than we can do. As this is a advanced use-case (migrating from local to remote), I am not quite sure if we should provide such a command.

Reply:
then remove it